_episodes/02-sql-aggregation.md

```diff
@@ -213,7 +213,7 @@ Using a view we will be able to access these results with a much shorter notatio
 
     SELECT *
     FROM summer_2000
-    WHERE species_id == 'PE';
+    WHERE species_id = 'PE';
```

Comment:
Good catch, can't believe we had those. SQLite seems to accept `==` which comes as a surprise to me.